xbmc/video/VideoInfoTag.cpp

```diff
@@ -998,9 +1065,32 @@ void CVideoInfoTag::SetPictureURL(CScraperUrl &pictureURL)
   m_strPictureURL = pictureURL;
 }
 
-void CVideoInfoTag::SetVotes(std::string votes)
+void CVideoInfoTag::AddRating(float rating, int votes, std::string type)
```

Comment:
Please specify the default value (same below): `const std::string& type /* = "" */`